src/components/pick-list/shared-list-tests.ts

```diff
@@ -201,6 +201,31 @@ export function keyboardNavigation(listType: ListType): void {
 
       expect(await getFocusedItemValue(page)).toEqual("two");
     });
+
+    it("resets tabindex to selected item when focusing out of list", async () => {
+      const page = await newE2EPage({
+        html: `
+        <calcite-${listType}-list>
+          <calcite-${listType}-list-item value="one" label="One" selected></calcite-${listType}-list-item>
+          <calcite-${listType}-list-item value="two" label="Two"></calcite-${listType}-list-item>
+        </calcite-${listType}-list>
+        <calcite-button name="next-element">next element</calcite-button>
```

Comment:
Can you update the test the following way?

1. create HTML page with list HTML only (no `#next-element`)
2. tab twice, assert on `activeElement` (should be `body`)
3. tab back, assert on first list item.